keyvault/data-plane/azure-keyvault-cryptography/src/main/java/com/microsoft/azure/keyvault/cryptography/SymmetricKey.java

```diff
@@ -315,7 +315,7 @@ public String getKid() {
         ICryptoTransform transform = null;
 
         try {
-            transform = algo.createEncryptor(key, null, provider);
+            transform = algo.createEncryptor(this.key, null, provider);
```

Comment:
This is probably wrong - using `this.key` means you're using the `key` from the class field, rather than the `key` passed into the method, which is changing the semantics here.

Reply:
The tests were failing before I made this change.  The original source was also using the class variable.
https://github.com/Azure/azure-sdk-for-java/blob/master/keyvault/data-plane/azure-keyvault-cryptography/src/main/java/com/microsoft/azure/keyvault/cryptography/SymmetricKey.java#L318

Reply:
Ah, I see! +1 then :-)